test/testcafe/framework/page-objects/SwitchOVEnrollChannelPageObject.js

```diff
@@ -20,6 +20,11 @@ export default class SwitchOVEnrollPageObject extends BasePageObject {
     await this.form.selectRadioButtonOption(channelOptionFieldName, index);
   }
 
+  async isFirstRadioButtonAutoSelected(ovFlowIsQrcode = true) {
```

Comment:
rename the fn to be flow agnostic, so it can support flows other than sms/qrcode if any will be added from backend in the future.
something like:
```
async isRadioButtonChecked(value: string)
```